.github/workflows/ci.yml

```
@@ -87,6 +87,9 @@ jobs:
       - name: Docker compose - integration tests
         if: ${{ matrix.testDockerCompose }}
         run: go run . integration-test
+        env:
+          # TODO(antlai-temporal): Remove this flag once server 1.27 released.
+          DISABLE_WORKER_DEPLOYMENT_TESTS: "1"
```

```suggestion
          DISABLE_SERVER_1_27_TESTS: "1"
```

@rodrigozhou will also need this flag to test attaching nexus callbacks I believe

---

Adding the flag